sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -96,6 +96,15 @@ abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
    */
   val childExprs: Seq[BaseExprMeta[_]]
 
+  /**
+   * If the count of `childExprs` is dynamic.
+   * E.g.: `Invoke` is a dynamic expression and can have different children
+   * Refer to the `children` function in `Invoke`:
+   *   // the arguments length is dynamic
+   *   override def children: Seq[Expression] = targetObject +: arguments
+   */
+  def isChildExprsCountDynamic: Boolean = false
```

Comment:
I saw this is designed only for the type checks, personally a better way is to introduce a new check class to handle this special behavior. We are using similar solution for CaseWhen and Cast.  You can refer to `CastChecks` and `CaseWhenChecks`.